alacritty_terminal/src/term/mod.rs

```diff
@@ -2219,6 +2226,18 @@ impl<T: EventListener> Handler for Term<T> {
             self.set_title(popped);
         }
     }
+
+    #[inline]
+    fn text_area_size_pixels<W: io::Write>(&mut self, writer: &mut W) {
+        let width = self.size.cell_width * self.grid.cols().0 as f32;
+        let height = self.size.cell_height * self.grid.screen_lines().0 as f32;
+        let _ = write!(writer, "\x1b[4;{};{}t", height, width);
+    }
+
+    #[inline]
+    fn text_area_size_chars<W: io::Write>(&mut self, writer: &mut W) {
+        let _ = write!(writer, "\x1b[8;{};{}t", self.grid.screen_lines(), self.grid.cols());
+    }
```

Comment:
Just to clarify on my previous comment: I'd be perfectly fine with a test that just calls these methods directly, I think that would make some sense.

But since the other handler methods also aren't really covered and the functionality is relatively simple, I wouldn't require it here. But since you had a test previously anyways, we could add it here.